packages/types/src/datatype/NodeId.ts

```diff
@@ -32,11 +32,9 @@ export namespace NodeId {
      */
     export const UNSPECIFIED_NODE_ID = NodeId(0);
 
-    export const toHexString = (nodeId: NodeId) => {
-        const writer = new DataWriter();
-        writer.writeUInt64(nodeId);
-        return Bytes.toHex(writer.toByteArray()).toUpperCase();
-    };
+    export function strOf(nodeId: NodeId) {
+        return hex.fixed(nodeId, 16);
```

Comment:
it could maybe be good to leave the "toUpperCase" here ... also logging wise maybe a good idea to have that ID which is "usually used uppercase" also in logs always uppercase (instead of "just another lowercase hex number" :-) ) Just for easier visiblity